Argument Parser

Add argument parsing functionality to main.rs.

something wrong. when run cargo run used clap v4.4.8
````
error[E0432]: unresolved import `clap::App`
 --> src/main.rs:1:12
  |
1 | use clap::{App, Arg};
  |            ^^^ no `App` in the root

error[E0599]: no method named `about` found for struct `Arg` in the current scope
  --> src/main.rs:12:18
   |
11 | /             Arg::new("input")
12 | |                 .about("Input file")
   | |                 -^^^^^ method not found in `Arg`
   | |_________________|
   | 

error[E0599]: no method named `about` found for struct `Arg` in the current scope
  --> src/main.rs:18:18
   |
17 | /             Arg::new("output")
18 | |                 .about("Output file")
   | |                 -^^^^^ method not found in `Arg`
   | |_________________|
   | 

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `rtgitee` (bin "rtgitee") due to 3 previous errors
````

````
error[E0599]: no function or associated item named `new` found for struct `Arg` in the current scope
  --> src/main.rs:11:18
   |
11 |             Arg::new("input")
   |                  ^^^ function or associated item not found in `Arg<'_, '_>`

error[E0599]: no function or associated item named `new` found for struct `Arg` in the current scope
  --> src/main.rs:17:18
   |
17 |             Arg::new("output")
   |                  ^^^ function or associated item not found in `Arg<'_, '_>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `rtgitee` (bin "rtgitee") due to 2 previous errors
````

````
error[E0599]: no method named `about` found for struct `Arg` in the current scope
  --> src/main.rs:12:18
   |
11 | /             Arg::with_name("input")
12 | |                 .about("Input file")
   | |                 -^^^^^ method not found in `Arg<'_, '_>`
   | |_________________|
   | 

error[E0599]: no method named `about` found for struct `Arg` in the current scope
  --> src/main.rs:18:18
   |
17 | /             Arg::with_name("output")
18 | |                 .about("Output file")
   | |                 -^^^^^ method not found in `Arg<'_, '_>`
   | |_________________|
   | 

For more information about this error, try `rustc --explain E0599`.
error: could not compile `rtgitee` (bin "rtgitee") due to 2 previous errors
````

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+clap = "2.33.0"

--- a/src/main.rs
+++ b/src/main.rs
@@ -1,3 +1,34 @@
+use clap::{App, Arg};
+
 fn main() {
-    println!("Hello, world!");
+    // Create a new instance of the command-line application
+    let app = App::new(env!("CARGO_PKG_NAME"))
+        .version(env!("CARGO_PKG_VERSION"))
+        .author(env!("CARGO_PKG_AUTHORS"))
+        .about(env!("CARGO_PKG_DESCRIPTION"))
+        // Define the command-line arguments
+        .arg(
+            Arg::with_name("input")
+                .help("Input file")
+                .index(1)
+                .required(true),
+        )
+        .arg(
+            Arg::with_name("output")
+                .help("Output file")
+                .index(2)
+                .required(true),
+        );
+
+    // Parse the command-line arguments
+    let matches = app.get_matches();
+
+    // Access the values of the arguments
+    let input_file = matches.value_of("input").unwrap();
+    let output_file = matches.value_of("output").unwrap();
+
+    // Your application logic goes here
+
+    println!("Input file: {}", input_file);
+    println!("Output file: {}", output_file);
 }

